examples/kubeflow/mnist_with_tfjob/README.md

```diff
@@ -0,0 +1,194 @@
+## Step 1 - Install and deploy both Kubeflow and Pachyderm
+Part of what makes [Pachyderm](https://pachyderm.com/) and [Kubeflow](https://www.kubeflow.org/) work so well together is that they're built on [Kubernetes](https://kubernetes.io/), which means they can run virtually anywhere. While both Pachyderm and Kubeflow have their own deployment instructions for various infrastructures, this instructional will be based on my personal favorite, GKE. Before continuing, make sure you have the following installed on your local machine. 
+
+#### Prerequisites:
+  - [Pachyderm cli](http://docs.pachyderm.com/en/latest/getting_started/local_installation.html#install-pachctl) 
+  - [Kubeflow cli](https://www.kubeflow.org/docs/started/getting-started/#installing-command-line-tools)
+  - [Kubectl cli](https://kubernetes.io/docs/tasks/tools/install-kubectl/)  
+  - [gcloud cli](https://cloud.google.com/sdk/gcloud/) 
+  - [docker](https://docs.docker.com/install/)
+
+#### Deploy:
+To make it simple, we created a simple [bash script](github.com/pachyderm/pachyderm/examples/kubeflow/mnist_with_tfjob/gcp-kubeflow-pachyderm-setup.sh) specifically for this post, and you can use it to deploy Pachyderm and Kubeflow together on GKE in no time. However, If you prefer to do this all on your local machine, or any other infrastructure, please refer to the links below.
+
+  - [Pachyderm Install Docs](http://docs.pachyderm.com/en/latest/getting_started/local_installation.html)
+  - [Kubeflow Install Docs](https://www.kubeflow.org/docs/started/getting-started/#installing-kubeflow)
+
+#### Working setup check
+1. `kubectl get pods -n kubeflow` returns running pods.
+2. `pachctl version` returns *both* pachctl and pachd versions.
+3. `pachctl enterprise get-state` returns: `Pachyderm Enterprise token state: ACTIVE` (If it doesn't, [Register Pachyderm](http://docs.pachyderm.com/en/latest/enterprise/deployment.html#activate-via-the-dashboard)) 
+
+## Step 2 - Checking in our data 
+Just like every other data science project, we begin by getting the data. In a blank directory on your local machine run:  
```

Comment:
I think you can structure this section as a numbered list with a little intro text:
Every data science project starts with loading your data into Pachyderm. To do so, you need to download a `mnist.npz` dataset to your local machine, create a Pachyderm repo, and then upload the dataset to the Pachyderm repo.

Complete the following steps:

1. Download the `mnist.npz` file to a black directory on your local machine:

   ```
   ➜ curl -O https://storage.googleapis.com/tensorflow/tf-keras-datasets/mnist.npz
   ```

1. Create a Pachyderm repo called `inputrepo`:

   ```
   $ pachctl create repo inputrepo
   ```

1. Add `mnist.npz` to `inputrepo`:

   ```
   $ pachctl put file inputrepo@master:/data/mnist.npz -f mnist.npz
   ```
    This command copies the `minst` dataset from your local machine to your Pachyderm repo `inputrepo`. This operation automatically generates a commit ID. Congratulations! Your data now has a `HEAD` commit, and Pachyderm has begun version-controlling the data!

1. Verify your configuration by running the following command:

    ```
    $ pachctl list file outputrepo@master:/data/ --history all
    COMMIT                           NAME            TYPE COMMITTED    SIZE     
    a1a45ecc348a4e41bfddb2ce32df1475 /data/mnist.npz file 1 minute ago 10.96MiB
    ```
    The `--history` flag displays the detail information about the commit.

1. Proceed to the next step.